### DISCUSSION TITLE
Downloading in a lower quality than available.

### DISCUSSION THREAD

**Discussion by Sean-Burnett** [Avnsx/fansly#87] [Q&A] [answered]
Currently experiencing an issue with one creator Akaneararagi where 1080+ is available on the video but it's downloading in 720. This doesn't occur for all videos. Doesn't appear to be any specific length or date range as some will download in 1080P but others of similar length and age will not. Notably the 4 most recent videos download in 720 but the 5th - 12 download in 1080 before it goes into 720 again for a few videos. Also occurs with messages. I'm using the most recent version of Windows 11, Chrome is up to date, attempted removal and redownload of scraper and issue persist. Any suggestions or wait out an eventual update?

**Comment by Avnsx** [comment, marked as answer]
Already know about this and it will get fixed with the version 0.4 which I will release within next couple days.

Related: https://github.com/Avnsx/fansly/issues/80
